### ISSUE TITLE
cmd 'code-server --install-exntension xxx.vsix' dosn't work.

### ISSUE THREAD

**Issue by spcidealacm** [cdr/code-server#2083]
![image](https://user-images.githubusercontent.com/8594876/92860667-dad71d00-f43b-11ea-8771-6d9403655e09.png)
as you see cannot install extension through the command line. please fix it.

**Comment by spcidealacm** [comment]
![image](https://user-images.githubusercontent.com/8594876/92862427-0c50e800-f43e-11ea-9053-7e12a9b2c0d5.png)
And even put it into the marketplace still cannot work.

**Comment by code-asher** [comment]
Sorry about this, we have a bug where installing extensions doesn't work when doing it inside code-server's terminal. Will fix for 3.5.1.

**Comment by code-asher** [comment]
One workaround for now is to run the command outside of code-server or to unset the socket environment variable (code-server is trying to use this to open a file in the existing instance):

```
VSCODE_IPC_HOOK_CLI= code-server --install-extension <id>
```

**Comment by code-asher** [comment]
One workaround for now is to run the command outside of code-server or to unset the socket environment variable (code-server is trying to use this to open a file in the existing instance):

```
VSCODE_IPC_HOOK_CLI= code-server --install-extension <id>
```

**Comment by nhooyr** [comment]
Merged fix into master. Will be fixed in v3.5.1
